src/js/profile/chart.js

```diff
@@ -140,7 +142,8 @@ export class Chart extends Observable {
                 }[index];
                 self.triggerEvent(`profile.chart.download_${downloadFn['type']}`, self);
 
-                downloadFn.fn(self.title);
+                let fileName = self.selectedFilter === null || self.selectedGroup === null ? `${self.title}` : `${self.title} - by ${self.selectedGroup} - ${self.selectedFilter}`;
```

Comment:
shouldn't this be just `self.selectedGroup` because `selectedFilter` will be set by selecting the group and be default to `All Values` right?

Reply:
removed `self.selectedFilter`